doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/decision-tables-defining-proc.adoc

```diff
@@ -30,7 +30,19 @@ NOTE: As an alternative to populating both the object type and constraint cells,
 If you need to add more rule tables to this decision table spreadsheet, skip a row after the last rule in the previous table, label another `RuleTable` cell in the same column as the previous `RuleTable` and  `RuleSet` cells, and create the new table following the same steps in this section (rows 19-29 in example).
 . Save your XLS or XLSX spreadsheet to finish.
 
-NOTE: Only the first worksheet in a spreadsheet workbook will be processed as a decision table when you upload the spreadsheet in {CENTRAL}. Each `RuleSet` name combined with the `RuleTable` name must be unique across all decision table files in the same package.
+[NOTE]
+====
+
+Only the first worksheet in a spreadsheet workbook is processed as a decision table when you upload the spreadsheet in {CENTRAL}. Each `RuleSet` name combined with the `RuleTable` name must be unique across all decision table files in the same package.
```

Comment:
Suggestion - clarify pairing:

s/Each `RuleSet` name combined with the `RuleTable` name must be unique across all decision table files in the same package./Each `RuleSet` and `RuleTable` name combination must be unique across all decision table files in the same package.

Reply:
@csherrar Here, we are referring only to the `RuleSet` name, but not the combination. So I believe it is fine as is. 
wdyt?